plugins/jobs/src/js/jobsToggleSchedule.js

```diff
@@ -0,0 +1,22 @@
+import { updateSchedule } from "#SRC/js/events/MetronomeClient";
+
+export default function jobsToggleSchedule(job) {
+  const [schedule] = job.schedules;
+  const isEnabled = schedule ? schedule.enabled : false;
+
+  const label = isEnabled ? "Disable Schedule" : "Enable Schedule";
+
+  const data = Object.assign({}, schedule, { enabled: !isEnabled });
+
+  return {
+    label,
+    onItemSelect() {
+      // TODO - use graphql here https://jira.mesosphere.com/browse/DCOS-37617
+      // take(1) makes sure the observable is going to complete after finishing
+      // the request, so we don't have to care about unsubscribing.
+      updateSchedule(job.id, data)
+        .take(1)
+        .subscribe();
```

Comment:
nothing? what about error handling? 🤔

Reply:
There was no errors being displayed in the previous implementation. If you click on a menu action that fails, you just do not see the results of it reflected on the UI.